MAINTAINERS.md

```diff
@@ -172,6 +172,8 @@ An automated list can be found [here](https://github.com/orgs/tldr-pages/people)
   [3 April 2024](https://github.com/tldr-pages/tldr/issues/12594) — present
 - **Magrid0 ([@Magrid0](https://github.com/Magrid0))**:
   [3 May 2024](https://github.com/tldr-pages/tldr/issues/12717) — present
+- **Machiavelli ([@MachiavelliII](https://github.com/MachiavelliII))**:
+  [27 Mar 2025](https://github.com/tldr-pages/tldr/issues/16036) -present
```

Comment:
This should be at the tail end of active org members

Reply:
It is
![image](https://github.com/user-attachments/assets/24ce67d9-2070-4cb5-a630-170508ba92dc)